proto-gen.sh

```diff
@@ -8,6 +8,6 @@ for i in "$PROTO_SRC"/*; do
 
 	if ls $i/*.proto > /dev/null 2>&1; then
 			echo "generating from $i"
-			protoc -I$i --go_out=$i $i/*.proto
+			protoc -I$i --gofast_out=$i $i/*.proto
```

Comment:
Would it be worth try other options, e.g., `gogofast` etc?

Reply:
Ah sorry, just saw this. I chose `--gofast_out` primarily to be compatible with [m3coordinator](https://github.com/m3db/m3coordinator/blob/master/generated/proto/prometheus/genproto.sh#L7:10). cc: @nikunjgit - just in case you did some deeper digging into this than I did.